src/widgets/index/index.ts

```diff
@@ -53,10 +90,18 @@ const index = (props: IndexProps): Index => {
   return {
     $$type: 'ais.index',
 
+    getIndexId() {
+      return indexName;
+    },
+
     getHelper() {
       return helper;
     },
 
+    getDerivedHelper() {
+      return derivedHelper;
```

Comment:
Do we really want to give an access to the `derivedHelper` instance? Its usage is mostly internal I don't think we have a use case to access its instance. We could only expose the results at the index level.